### PR TITLE
[docs] Adding import xls documentation

### PR DESCRIPTION
**Pull request by Dimfacion**

### CODE REVIEW

File: docs/usage/scenario_import.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+![List of Mappers](assets/mapper_screen.png)
+
+### Setting up the mapper to tell each injects apart 
+
+When creating a new mapper, you will quickly be asked to choose an inject type column. This column is the one that will allow the mapper to figure out which injects to create (Sending a mail, sending an sms, ...). Once this column has been chosen, you can add a representation for an inject type. 
+
+The first thing to define in this representation is the matching type in the xls. This is the value that will define which inject to create when scanning the column defined in "inject type column". For instance, if you want to create an inject that sends individual emails when in the column there is the word "mail", then you will need to set the value as "mail". You can choose a word that has nothing to do with the inject type and you can also make use of [regular expressions](https://en.wikipedia.org/wiki/Regular_expression) in this field. Please keep in mind though that this value is case sensitive.
```
Comment (RomuDeuxfois) [inline docs/usage/scenario_import.md:15]: I am not sure to understand that part

"You can choose a word that has nothing to do with the inject type"

I understand what you mean but it seems confusing when I read it. Maybe iy's better to remove it and let just the regular explanation

Comment (Dimfacion) [reply docs/usage/scenario_import.md:15]: Yeah, I think you're right, I added this phrase in an earlier version where it wasn't very clear but should have removed it when I reworked this part.

File: docs/usage/scenario_import.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Importing Injects into a Scenario
+
+Recreating [injects](injects.md) that were already defined in a spreadsheet can be a frustrating task. To help users save time, we added the possibility to import injects as defined in an xls file into a [scenario](scenario.md). This is done via a two-steps process : [creating a mapper](#how-to-create-a-mapper) and [importing the injects using the mapper](#how-to-import-injects-into-a-scenario-using-a-mapper).
```
Comment (Jipegien) [inline docs/usage/scenario_import.md:3]:
```suggestion
Recreating a timeline of [injects](injects.md) that were already defined in a spreadsheet can be a frustrating task. To help users save time, we added the possibility to import injects defined in an xls file into a [scenario](scenario.md). This is done via a two-steps process : [creating a mapper](#how-to-create-a-mapper) and [importing the xls file using the mapper](#how-to-import-injects-into-a-scenario-using-a-mapper).
```

File: docs/usage/scenario_import.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+| h         | clock-hour-of-am-pm (1-12) | number   | 12        |
+| H         | hour-of-day (0-23)    | number        | 0         |
+| m         | minute-of-hour        | number        | 30        |
+| s         | second-of-minute      | number        | 55        |
+| '         | escape for text       | delimiter     |           |
+| ''        | single quote          | literal       | '         |
+| [         | optional section start|               |           |
+| ]         | optional section end  |               |           |
+
+Please, do note that if you wish to use only hour of the day (e.g. 9:30) in your trigger time, you need to set a launch date on your scenario.
```
Comment (Jipegien) [inline docs/usage/scenario_import.md:39]:
```suggestion
Please, do note that if you wish to use exact time of the day (e.g. 9:30) in your trigger time, you will need to set a launch date on your scenario before importing the timeline of injects from the xls file.
```

File: docs/usage/scenario_import.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+| Symbol    | Meaning               |
+|-----------|-----------------------|
+| D         | Day                   |
+| J         | Day                   |
+| H         | Hour                  |
+| T         | Hour                  |
+| M         | Minutes               |
+
+This means that if you want your inject to start 2 days, 3 hours and 45 minutes after the start of your scenario, you can set the trigger time to D+2 H+3 M+45. When using relative dates, keep in mind that no time pattern need to be defined in the trigger time field.
```
Comment (Jipegien) [inline docs/usage/scenario_import.md:51]:
```suggestion
This means that if you want your inject to start 2 days, 3 hours and 45 minutes after the start of your scenario, you can set the trigger time to D+2 H+3 M+45. When using relative dates, you do not need to define a pattern for the trigger time field.
```

File: docs/usage/scenario_import.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+Once you have set all the fields you wish to set, you can click on the create button if you wish to create your mapper but you can also click on the test button to check that it works as intended.
```
Comment (Jipegien) [inline docs/usage/scenario_import.md:53]:
```suggestion
Once you have set all fields you wish to set, you can click on the create button if you wish to create your mapper but you can also click on the test button to check that it works as intended.
```

**Review by Jipegien — CHANGES_REQUESTED** [review]

**Review by Jipegien — APPROVED** [review]

**Pull request merged**